### DISCUSSION TITLE
想问一下作者的测试电脑配置

### DISCUSSION THREAD

**Discussion by MHBZHY** [CHNZYX/Auto_Simulated_Universe#248] [General]
我的电脑是AMD Ryzen 5800X3D + AMD Radeon 7900XTX，近几个版本几乎没有正常运行过。。。大部分都是在两到三层就离开了，很少看到最后boss。想问下作者是否仅在intel和nvidia平台上测试过。并没有要求作者适配AMD平台的意思，只想确认下是否果真如此，如果是的话我就去买一个IN平台来玩。

**Comment by MHBZHY** [comment]
基本上挂一个下午，连沉浸器都用不完

**Comment by CHNZYX** [comment]
和平台没有关系，遇到问题先看Q&A

**Reply by MHBZHY** [reply]
破案了……是因为我开启了amd显卡驱动里面的虚拟分辨率导致的，我关闭之后确实表现好了一些
